platform/windows_platform_test.go

```diff
@@ -269,15 +269,17 @@ var _ = Describe("WindowsPlatform", func() {
 	})
 
 	Describe("GetDefaultNetwork", func() {
-		It("delegates to the defaultNetworkResolver", func() {
-			defaultNetwork := boshsettings.Network{IP: "1.2.3.4"}
-			fakeDefaultNetworkResolver.GetDefaultNetworkNetwork = defaultNetwork
-
-			network, err := platform.GetDefaultNetwork()
-			Expect(err).ToNot(HaveOccurred())
-
-			Expect(network).To(Equal(defaultNetwork))
-		})
+		for _, value := range []bool{true, false} {
```

Comment:
Same as above `value` should have a name that expresses intent, ideally matching the name in the signature of `GetDefaultNetwork()`